CUDLR/Scripts/Console.cs

```diff
@@ -14,7 +15,25 @@ struct QueuedCommand {
   }
 
   public class Console {
-
+     
+     /* Struct containing console message and color data */
+	 struct ConsoleMessage
+      {
+          public string message;
+          public Color32 color;
```

Comment:
Sounds good to me, I'm not sure what other classes can interact with this but as it does have implicit conversion I think consistency is better than anything else here, and the justification you have for using C32 over C has convinced me.